games-strategy/wesnoth/wesnoth-1.18.2.recipe

```diff
@@ -0,0 +1,130 @@
+only limited by your creativity — make your own custom units, create your own \
+maps, and write your own scenarios or even full-blown campaigns. You can also \
+challenge up to eight friends—or strangers—and fight in epic multiplayer \
+fantasy battles."
+HOMEPAGE="http://wesnoth.org/"
+COPYRIGHT="2003-2024 The Battle for Wesnoth Project"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://sourceforge.net/projects/wesnoth/files/wesnoth-1.18/wesnoth-$portVersion/wesnoth-$portVersion.tar.bz2"
+CHECKSUM_SHA256="5f9f6d6105d48cf614eb06cf72273d62931dcb5d4982d75e851c3fa2fa70553a"
+PATCHES="wesnoth-$portVersion.patchset"
+ADDITIONAL_FILES="wesnoth.rdef.in"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	wesnoth$secondaryArchSuffix
+	cmd:wesnoth
+	cmd:wesnothd
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libboost_filesystem$secondaryArchSuffix
+	lib:libboost_iostreams$secondaryArchSuffix
+	lib:libboost_locale$secondaryArchSuffix
+	lib:libboost_program_options$secondaryArchSuffix
+	lib:libboost_regex$secondaryArchSuffix
+	lib:libboost_system$secondaryArchSuffix
+	lib:libboost_graph$secondaryArchSuffix
+	lib:libboost_coroutine$secondaryArchSuffix
+	lib:libcairo$secondaryArchSuffix
+	lib:libfontconfig$secondaryArchSuffix
+	lib:libpangocairo_1.0$secondaryArchSuffix
+	lib:libSDL2_2.0$secondaryArchSuffix
+	lib:libSDL2_image_2.0$secondaryArchSuffix
+	lib:libSDL2_mixer_2.0$secondaryArchSuffix
+	lib:libvorbisfile$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libdbus_1$secondaryArchSuffix # Optional, for notification support
+	lib:libicui18n$secondaryArchSuffix
+	lib:libicudata$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libharfbuzz$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libgobject_2.0$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	cairo1.18${secondaryArchSuffix}_devel # do not use xcairo
+	devel:libboost_filesystem$secondaryArchSuffix # >= 1.70.0
+	devel:libboost_iostreams$secondaryArchSuffix # >= 1.70.0
+	devel:libboost_locale$secondaryArchSuffix # >= 1.70.0
+	devel:libboost_program_options$secondaryArchSuffix #> = 1.70.0
+	devel:libboost_regex$secondaryArchSuffix # >= 1.70.0
+	devel:libboost_system$secondaryArchSuffix # >= 1.70.0
+	devel:libboost_graph$secondaryArchSuffix
+	devel:libboost_coroutine$secondaryArchSuffix
+	devel:libfontconfig$secondaryArchSuffix
+	devel:libpangocairo_1.0$secondaryArchSuffix
+	devel:libSDL2_2.0$secondaryArchSuffix
+	devel:libSDL2_image_2.0$secondaryArchSuffix
+	devel:libSDL2_mixer_2.0$secondaryArchSuffix
+	devel:libvorbisfile$secondaryArchSuffix
+	openssl3_devel$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	devel:libdbus_1$secondaryArchSuffix # Optional, for notification support
+	gcc_syslibs_devel$secondaryArchSuffix
+	gettext_devel$secondaryArchSuffix
```

Comment:
```suggestion
	lib:libintl$secondaryArchSuffix
```

Reply:
After setting it like this cmake couldn't find gettext anymore, solved it by moving it into BUILD_PREREQUIRES as cmd:gettext

Reply:
That's because it's not searching for gettext, it needs it's binaries in BUILD_PREREQUIRES:
```
cmd:msgfmt
cmd:msgmerge
```

Reply:
Specifying it as devel:libgettextlib works as well

Reply:
cmake chekcs for the commans, please resort to those.